lib/fakeroot.py

```diff
@@ -208,9 +208,9 @@
    #      dpkg-query -Wf '${Package}\n' \
    #      | egrep '^(fakeroot|fakeroot-ng|pseudo)$'
 
-   "debSB":
-   { "name": "Debian 9 (Stretch) or 10 (Buster)",
-     "match": ("/etc/debian_version", r"^(9|10)\."),
+   "debdistro":
```

Comment:
```suggestion
   "debian":
```

Reply:
I'm not 100% sold on these but those are the suggestions I had. Maybe another is `debderiv`.

Reply:
I like `debderiv` I think